Makefile

```diff
@@ -279,26 +279,26 @@ oci-build: manifests generate fmt vet test-envtest
 # To properly provided solutions that supports more than one platform you should use this option.
 .PHONY: oci-buildx
 oci-buildx: manifests generate fmt vet test-envtest ## Build OCI image for the manager for cross-platform support
-ifeq ($(IMAGE_BUILDER), docker)
 # copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
 	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
+ifeq ($(IMAGE_BUILDER), docker)
 	- $(IMAGE_BUILDER) buildx create --name project-v3-builder
 	$(IMAGE_BUILDER) buildx use project-v3-builder
 	- $(IMAGE_BUILDER) buildx build --push --platform=$(PLATFORMS) --tag $(OPERATOR_IMG) -f Dockerfile.cross .
 	- $(IMAGE_BUILDER) buildx rm project-v3-builder
-	rm Dockerfile.cross
 else ifeq ($(IMAGE_BUILDER), podman)
 	for platform in $$(echo $(PLATFORMS) | sed "s/,/ /g"); do \
 		os=$$(echo $${platform} | cut -d/ -f 1); \
 		arch=$$(echo $${platform} | cut -d/ -f 2); \
-		BUILDAH_FORMAT=docker $(IMAGE_BUILDER) buildx build --manifest $(OPERATOR_IMG) --platform $${platform} --build-arg TARGETOS=$${os} --build-arg TARGETARCH=$${arch} . ; \
+		BUILDAH_FORMAT=docker $(IMAGE_BUILDER) buildx build -f Dockerfile.cross --manifest $(OPERATOR_IMG) --platform $${platform} --build-arg TARGETOS=$${os} --build-arg TARGETARCH=$${arch} . ; \
```

Comment:
Do we need this `--platform` argument and Dockerfile.cross when building with Podman?

Reply:
I think the `Docker.cross` just ensures that we are using cross compilation with Golang. But that would mean we don't need qemu. I will check...

Reply:
@andrewazores did we need this `--platform` for other multi-arch builds or is this just a Docker thing?

Reply:
The other multiarch builds are mostly using `buildah`, in particular through a GH Action that abstracts over it, but I think the `--platform` makes sense when using the builders directly like this.